cluster_tools/dockered-slurm/slurm.conf

```diff
@@ -36,7 +36,7 @@ ReturnToService=0
 #SrunEpilog=
 #TaskProlog=
 #TaskEpilog=
-#TaskPlugin=
+TaskPlugin=task/affinity
```

Comment:
can you add a comment that explains why this is set?

Reply:
I added a comment and also added an assertion to the test, checking that the plugin is enabled.